cura/Settings/MachineManager.py

```diff
@@ -862,6 +858,19 @@ def correctExtruderSettings(self) -> None:
                 title = catalog.i18nc("@info:title", "Settings updated"))
             caution_message.show()
 
+    def _correctPrintSequence(self) -> None:
```

Comment:
It would help to document why Print Sequence is a special case here. Won't this also happen with other similar settings? Like with all enum-settings, or with all global settings or something?

Reply:
Better?

Reply:
Fine by me.
Clear documentation through out the code